js/shared.js

```diff
@@ -877,6 +877,7 @@ export const questionnaireModules = () => {
             'Smoking, Alcohol, and Sun Exposure': {path: 'module3Stage.txt', moduleId:"Module3", enabled:false},
             'Where You Live and Work': {path: 'module4Stage.txt', moduleId:"Module4", enabled:false},
             'Enter SSN': {path: 'ssnModule.txt', moduleId:"ModuleSsn", enabled:false},
+            'Covid-19': {path: 'mockSurvey.txt', moduleId:"ModuleCovid19", enabled:true},
```

Comment:
should we update the name of this file to `covid-19Survey.txt` or `covid-19Stage.txt` or something similar? When I see 'mock' I think it's used for testing or a placeholder.

Reply:
@JoeArmani thanks for your suggestion. You are correct, it's temp file since they just want to see the description of this survey on dashboard now. We have not had `covid-19Surver.txt` yet in repo https://github.com/episphere/questionnaire/blob/main/mockSurvey.txt.